templates/download/raspberry-pi/index.html

```diff
@@ -571,7 +571,7 @@ <h2>Make something great today</h2>
             height="150",
             hi_def=True,
             loading="lazy",
-            attrs={"class": "p-heading-icon__img--small"}
+            attrs={"class": "p-heading-icon__img p-heading-icon__img--small"}
```

Comment:
This one looks better in production. Seems to align with the title better.

Reply:
I know, but it needs the `p-heading-icon__img` classname that comes from Vanilla, `p-heading__img--small` is custom ubuntu.com class, and for reliable migration we can't rely just on it.

I'll see what I can do to improve the look.

Reply:
Just checked, and I don't see much difference TBH.

Prod:
<img width="1262" alt="image" src="https://user-images.githubusercontent.com/83575/233269717-e056a93a-d51e-40f6-ad7a-6911db52ae52.png">

Branch:

<img width="1223" alt="image" src="https://user-images.githubusercontent.com/83575/233269656-0f2620d7-219d-4fb9-8b97-3fc4c0b4fe5e.png">